src/deno_json/mod.rs

```diff
@@ -471,19 +507,14 @@ pub struct ConfigFileJson {
 
   pub name: Option<String>,
   pub version: Option<String>,
-  #[serde(alias = "workspaces")]
   pub workspace: Option<Vec<String>>,
+  #[serde(rename = "workspaces")]
+  pub(crate) deprecated_workspaces: Option<Vec<String>>,
```

Comment:
Do we have to keep this one? Are there any usages of it in the wild?

Reply:
Yeah, deno_std. I think it's good to have for when people use the wrong key.